CHANGELOG.md

```diff
@@ -8,6 +8,107 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Behavioral Twin: Model and Data processing (MDP) KIT](https://eclipse-tractusx.github.io/docs-kits/category/model-based-development-and-data-processing-mdp-kit) |                                                       n/a                                                       | [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20MDP%20Kit/Changelog%20Model%20Based%20Development%20and%20Data%20Processing%20Kit) |
+| [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/category/remaining-useful-life-rul-kit)                           |                                                       n/a                                                       |               [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20RuL%20Kit/Changelog%20Remaining%20Useful%20Life%20Kit)               |
+| [Circularity KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/page-adoption-view)                                                           |                                                       n/a                                                       |                            [0.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/Changelog%20Circularity%20KIT)                             |
+| [Modular Production KIT](https://eclipse-tractusx.github.io/docs-kits/category/modular-production-kit)                                                            |                                                       n/a                                                       |                  [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/Modular%20Production%20Kit/Modular%20Production%20Kit%20Changelog)                   |
+
+
+
+### Known knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+- Conformity to [Asset Administration Shell v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf), [API live example](https://app.swaggerhub.com/apis/Plattform_i40/AssetAdministrationShellRegistryServiceSpecification/V3.0_SSP-001#/Asset%20Administration%20Shell%20Registry%20API/GetAssetAdministrationShellDescriptorById) (AAS) is given. The DTR “/query” interface is finally deprecated and no longer available with this Release as announced. Details can be found in the [changelog](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/CHANGELOG.md) of the respective component.
```

Comment:
change wording to
Conformity to [Asset Administration Shell (AAS) API v3.0] (https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf) and bugfix v3.0.1 is given. Fine granular access control introduced for optional use.

Reply:
done